ginga/qtw/Widgets.py

```diff
@@ -341,6 +341,10 @@ def set_color(self, fg=None, bg=None):
         self.widget.setStyleSheet(
             "QLabel { background-color: %s; color: %s; }" % (bg, fg))
 
+    def set_halign(self, align):
+        # TODO: set horizontal alignment of text
+        pass
```

Comment:
Only that in the case of wrapped widgets, where it may be desirable to have the widget work even if it cannot 100% support the API functionality, IFF the effect is only cosmetic, it may be better to do a `pass`. However, in this case I think a Qt widget does support a horizontal alignment so will check into that.